package.json

```diff
@@ -24,26 +24,41 @@
   },
   "homepage": "https://github.com/tonyhb/redux-ui#readme",
   "devDependencies": {
-    "babel-cli": "6.5.1",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
+    "@babel/cli": "^7.0.0",
```

Comment:
I think we should remove '^" from versioning, to make the package more predictable.

Reply:
Good point, updated 👍